src/Microsoft.OData.Edm/Validation/EdmError.cs

```diff
@@ -21,17 +21,35 @@ public class EdmError
         /// <param name="errorCode">An integer code representing the error.</param>
         /// <param name="errorMessage">A human readable message describing the error.</param>
         public EdmError(EdmLocation errorLocation, EdmErrorCode errorCode, string errorMessage)
+         : this(errorLocation, errorCode, errorMessage, Severity.Undefined)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the EdmError class.
+        /// </summary>
+        /// <param name="errorLocation">The location where the error occurred.</param>
+        /// <param name="errorCode">An integer code representing the error.</param>
+        /// <param name="errorMessage">A human readable message describing the error.</param>
+        /// <param name="errorLevel">An enum value describing the error severity.</param>

```

Comment:
@wachugamaina Update name of this param from `errorLevel` to `severity`

Reply:
Resolved